src/app/game/buildings/buildings.service.ts

```diff
@@ -9,14 +9,18 @@ import { environment } from '../../../environments/environment';
 
 export class BuildingsService {
 
-  constructor(private http: HttpClient) { }
+  constructor(private http: HttpClient) {
+  }
 
   getBuildings(): Observable<any> {
-    return this.http.get('/game/buildings', { observe: 'response' });
+    const buildings =  this.http.get(environment.getBuildings, { observe: 'response' });
+    buildings.subscribe((response) => {
+      localStorage.setItem('buildings', JSON.stringify(response.body['buildings']));
+    });
```

Comment:
@HonzaDudek and the create building? the update building? checking the difference between backend state of buildings and localstorage?